website/views/pages/docs/basic-documentation.ejs

```diff
@@ -180,7 +180,7 @@
   
         </div>
 
-        <div purpose="content" id="body-content" class="d-flex flex-column p-0 px-lg-5 content">
```

Comment:
@eashaw thank you!  FYI this is related to https://github.com/mikermcneil/parasails/pull/4